lib/dgram.js

```diff
@@ -180,6 +180,7 @@ function replaceHandle(self, newHandle) {
 
   // Replace the existing handle by the handle we got from primary.
   oldHandle.close();
+  oldHandle = null;
```

Comment:
This isn't necessary since `oldHandle` isn't used in any child scopes.

Reply:
OK, fixed it. I just want to release it asap.